src/Nethermind/Nethermind.Merge.Plugin/Synchronization/PosForwardHeaderProvider.cs

```diff
@@ -67,8 +67,9 @@ private void TryUpdateTerminalBlock(BlockHeader currentHeader)
     // Used only in get block header in pre merge forward header provider, this hook stops pre merge forward header provider.
     protected override bool ImprovementRequirementSatisfied(PeerInfo? bestPeer)
```

Comment:
You should not need to care about this class. Does this cause a problem somehow?

Reply:
Nevermind, I misread.